### ISSUE TITLE
is this now deprecated in favor of Microsoft.AspNet.WebApi.Client

### ISSUE THREAD

**Issue by xtianus79** [Azure/azure-mobile-apps-net-client#468]
I am trying to follow a tutorial and this mobile app client isn't working and instead the project is saying to do this... Microsoft.AspNet.WebApi.Client

**Comment by guperrot** [comment]
Hi, what tutorial are you following?

**Comment by xtianus79** [comment]
@guperrot this one

https://docs.microsoft.com/en-us/azure/app-service-mobile/app-service-mobile-xamarin-forms-get-started

althought it seems to be back to normal now.... however, the tutorial is still not working

**Comment by guperrot** [comment]
Can you provide more details about what is not working for you? What step is failing, what error do you get, do you have logs to share?

**Comment by xtianus79** [comment]
@guperrot I have another post of something that is not working currently.  What is not working is from the above tutorial... that shows this step 
https://docs.microsoft.com/en-us/azure/includes/media/app-service-mobile-dotnet-backend-create-new-service/search-mobile-apps-quickstart.png

it wasn't showing those mobile choices... it just had 3 choices... those 3 were java, aspnet and node.  When choosing aspnet it said this... instead of what's below, which is not consistent with the tutorials it said to add this from nuget > Microsoft.AspNet.WebApi.Client

And that is what didn't work specifically this code 

> public class Contact
```
{
    public int Id { get; set; }
    public string Name { get; set; }
    public string EmailAddress { get; set; }
}
```
                    
> Then add this code to Program.cs (or whichever file you intend on augmenting) with this code, which reaches out and calls the API and returns the data so you can use it.


```
using (HttpClient client = new HttpClient())
{
    client.BaseAddress = new   
        Uri("https://travelrecordapp7.azurewebsites.net");
    
    var json = client.GetStringAsync("/").Result;
    var contacts = JsonConvert.DeserializeObject<contact[]>(json);
}
```


This is the correct information that suddenly changed back today on Azure. 
> Step 3 Configure your client application
CREATE A NEW APPCONNECT AN EXISTING APP
If using Xamarin Studio, right-click your project, select 'Add' and then 'Add NuGet Packages.' Search for the **Microsoft.Azure.Mobile.Client** package, and then click 'Add Package.'
If using Visual Studio, right-click your project and select 'Manage NuGet Packages' search for the Microsoft.Azure.Mobile.Client package, and click 'Install.'

**Comment by guperrot** [comment]
Hi, the screenshot seems outdated indeed.

If you are trying to download the quickstart for Xamarin.Forms then it's not related to ASP .NET which is server side.

To download the quick start for Xamarin.Forms application, **create or select an app service** in the portal then quickstart like in the following screenshot:

![capture](https://user-images.githubusercontent.com/623672/47397117-a7256980-d6e2-11e8-80fa-50697f946a21.PNG)

You will see the Xamarin.Forms quickstart code uses the right packages and has no contact class or anything related to ASP .NET.

**Comment by xtianus79** [comment]
@guperrot yes now it looks like what you are showing.  perhaps it was a glitch for a moment.  But now when i use the nuget package Microsoft.WindowsAzure.MobileServices it won't work when trying use CurrentPlatform it gives me an error saying is inaccessible due to its protection level.

**Comment by guperrot** [comment]
As discussed on https://github.com/Azure/azure-mobile-apps-net-client/issues/469#issuecomment-432712524 this API call is not needed on UWP and does not exist.

**Comment by xtianus79** [comment]
@guperrot to be fair this issue wasn't the same as the related post.  This issue was very clear. The choices on Azure were not the same as they are now.  To ask, was the Microsoft.AspNet.WebApi.Client package something that was deprecated in favor of the mobile client installation.  Whatever it is, it is fixed now.

**Comment by guperrot** [comment]
Hi, sorry I don' understand your question, we have not changed anything and AspNet client is not related to Mobile Services, Asp.net is a different product for developing web services.
